gym/envs/toy_text/taxi.py

```diff
@@ -237,7 +237,12 @@ def render(self, mode="human"):
             return self._render_gui(mode)
 
     def _render_gui(self, mode):
-        import pygame  # dependency to pygame only if rendering with human
```

Comment:
maybe this comment shouldn't be deleted, unless it's out of date

Reply:
Thanks, I deleted it because I thought it was kinda obvious but I imagine in a year or twos time it may not so I will change back